app/views/posts/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<section>
+  <h1 class="users-title">Add a new post</h1>
+  <%= form_with model: @post, url: user_posts_path, class: "card left" do |form| %>
+    <%= form.label :title %> <br>
+    <%= form.text_field :title %> <br>
+    <%= form.label :text %> <br>
+    <%= form.text_area :text , cols: 40, rows: 10 %> <br>
+    <%= form.submit "Submit", class: "btn" %>
+  <% end %>
```

Comment:
✅ There is a form in the views/posts/new.html.erb file which accepts input for creating posts 👍🏻